apps/financial-aid/backend/migrations/20210920105246-create-staff.js

```diff
@@ -0,0 +1,50 @@
+'use strict'
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction((t) =>
+      queryInterface.createTable(
+        'staff',
+        {
+          id: {
+            type: Sequelize.UUID,
+            primaryKey: true,
+            allowNull: false,
+            defaultValue: Sequelize.UUIDV4,
+          },
+          national_id: {
+            type: Sequelize.STRING,
+            allowNull: false,
+          },
+          name: {
+            type: Sequelize.STRING,
+            allowNull: false,
+          },
+          municipalityId: {
+            type: Sequelize.UUID,
+            allowNull: false,
+          },
+          role: {
+            type: Sequelize.ENUM('employee', 'admin'),
```

Comment:
Most of our enum have first letter higher case, should it not be here? 🤓

Reply:
This actually a type, but did some manipulation here to match that, but will change the base type to enum instead 😃